### ISSUE TITLE
organization follow menu issue V11.0.3

### ISSUE THREAD

**Issue by abdelazizelesh** [unacms/UNA#2982]
Allow to follow Org without joining Org if Org privacy is public, it's already possible to follow from timeline.
---

From studio i have checked the follow action is enabled for organization actions as below.
![image](https://user-images.githubusercontent.com/6819385/93718536-2a30f200-fb8e-11ea-92c6-7c1832b56400.png)
But in organization view page, the action is not rendering as below
![image](https://user-images.githubusercontent.com/6819385/93718568-4af94780-fb8e-11ea-936d-3315cc9bdf66.png)
Please adivce

**Comment by AlexTr** [comment]
I think follow is allowed after user is joined organization.

**Comment by abdelazizelesh** [comment]
@AlexTr thanks for your reply, i think this is a bug as i can do the follow from any organization post without joining the organization, please check below screenshot. i'm not a member nor friend and can follow from the post. 
![image](https://user-images.githubusercontent.com/6819385/93729862-4442f280-fbd7-11ea-8683-b625be353ebb.png)

**Comment by AlexTr** [comment]
Thank you @abdelazizelesh for pointing this out.

**Comment by Miroslav36** [comment]
Hello @AlexTr , @AntonLV ,
What is the choosen logic for the follow button ? Will it appear 1)once you've joined the organization or 2) is it visible for everyone (even if you have not joined the organization) ?
I have just updated to 11.0.4 and all "follow" buttons have disappeared, even the in the case mentionned by @abdelazizelesh , so it looks that you are in logic (1). I personally would have preferred logic (2) where anyone can follow the news of an organization...
Thank you
edit : I've just tested on a "fresh" install : you must join the organization, then the"unfollow" button is visible. Once you click on "unfollow", "follow" button will appear.
I wonder how I can create a "fan" page, where people can follow without joining...
edit2: maybe "favorite" could do the trick... will it generate identical notifications as the "follow" process ?
